altair/theme.py

```diff
@@ -0,0 +1,266 @@
+"""Customizing chart configuration defaults."""
+
+from __future__ import annotations
+
+from functools import wraps as _wraps
+from typing import TYPE_CHECKING
+
+from altair.vegalite.v5.schema._config import (
+    AreaConfigKwds,
+    AutoSizeParamsKwds,
+    AxisConfigKwds,
+    AxisResolveMapKwds,
+    BarConfigKwds,
+    BindCheckboxKwds,
+    BindDirectKwds,
+    BindInputKwds,
+    BindRadioSelectKwds,
```

Comment:
# NOTE
- I've manually copied these over for now.
- Would prefer this to using a `*` import
- May explore managing the declaration of this in a similar way to the `__all__` definition